easybuild/easyconfigs/c/configparser/configparser-3.5.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -12,26 +12,18 @@ toolchain = {'name': 'foss', 'version': '2016a'}
 source_urls = ['https://pypi.python.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/']
 sources = ['configparser-%(version)s.tar.gz']
 
-dependencies = [
-    ('Python', '2.7.11'),
-]
-builddependencies = [
-    ('pip', '8.1.2', versionsuffix),
-]
+patches = ['configparser-%(version)s_no-backports-namespace.patch']
+
+dependencies = [('Python', '2.7.11')]
+builddependencies = [('pip', '8.1.2', versionsuffix)]
```

Comment:
the one included in Python 2.7.11 is too old, pip 8.x is required for `pip install --prefix`